File: LogiOS/TrustarApp/Chat/ChatRoomWithUsers.swift
//
//  ChatRoomWithUsers.swift
//  TrustarApp
//
//  Created by DionSoftware on 2022/11/25.
//  Copyright © 2022 DionSoftware. All rights reserved.
//

import Foundation

class ChatRoomWithUsers {
    var room: ChatRoom
    var users: [ChatUser]
    init(room: ChatRoom, users: [ChatUser]) {
        self.room = room
        self.users = users
    }
}

File: LogAndroid/TrustarApp/src/main/kotlin/jp/co/toukei/log/trustar/chat/ChatRoomWithUsers.kt
package jp.co.toukei.log.trustar.chat

import jp.co.toukei.log.trustar.db.chat.entity.ChatRoom
import jp.co.toukei.log.trustar.db.chat.entity.ChatUser

class ChatRoomWithUsers(
    @JvmField val room: ChatRoom,
    @JvmField val users: List<ChatUser>,
)
